src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -163,6 +163,22 @@ function shouldAutoFocusHostComponent(type: string, props: Props): boolean {
   return false;
 }
 
+// TODO: Better polyfill
+let now;
+if (
+  typeof window !== 'undefined' &&
+  window.performance &&
+  typeof window.performance.now === 'function'
+) {
+  now = function() {
```

Comment:
`performance.now` needs to be bound, but looks like `Date.now` doesn't. I'll update.